neo/Network/P2P/RemoteNode.cs

```diff
@@ -63,6 +63,8 @@ private void EnqueueMessage(Message message)
                 case "getaddr":
                 case "getblocks":
                 case "getheaders":
+                case "getroots":
+                case "roots":
```

Comment:
`roots` should be the same as `headers`.

Reply:
If there are nodes don't upgrade to fix version, `getroots` will still be `not_single` and if `roots` is not `is_single`, it will still full the queue.

Reply:
https://github.com/neo-project/neo/blob/d29e310b6d0882bb3dc276d57da1f10121f71b02/neo/Network/P2P/ProtocolHandler.cs#L420-L435

You should modify here too.

Reply:
done